### DISCUSSION TITLE
Consulta sobre medios de pago

### DISCUSSION THREAD

**Discussion by webadapta** [fedealvz/Woo-iFactura#38] [Q&A]
Hola @fedealvz / comunidad,

Tengo un sitio que tiene MercadoPago para Argentina y PayPal para el exterior (en dólares). Necesitaría que el plugin sólo facture aquellos pedidos de MercadoPago, esto es posible configurarlo?

Gracias!

Germán

**Comment by villaboamiguel** [comment]
Hola German,

El plugin no tiene dicha funcionalidad, pero al ser de código abierto quizás puedas agregar el comportamiento antes de insertar el botón para emitir el comprobante en los detalles del pedido.

Saludos!
